mantidproject/mantid
# Update sans documentation

**Description of work.**
This PR updates the documentation for the ISIS SANS interface to bring it in line with the current state of the code. This is mainly updating images
**Report to:** Not applicable 



**To test:**

Check documentation changes for correctness and clarity.



Fixes no related issue. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
